app/assets/stylesheets/govuk_publishing_components/components/_modal-dialogue.scss

```diff
@@ -0,0 +1,138 @@
+$govuk-modal-margin: govuk-spacing(6);
+$govuk-modal-close-button-size: 40px;
+$govuk-modal-z-index: 1000;
+$govuk-modal-wide-breakpoint: $govuk-page-width + $govuk-modal-margin * 2 + $govuk-border-width-form-element * 2;
+
+.gem-c-modal-dialogue {
+  display: none;
+  position: fixed;
+  z-index: $govuk-modal-z-index;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  outline: 0;
+}
+
+.gem-c-modal-dialogue__box {
+  display: block;
+  position: fixed;
+  background: govuk-colour('white');
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  overflow-y: auto;
+  border: 0;
+
+  @include govuk-media-query($from: tablet) {
+    position: relative;
+    top: inherit;
+    right: inherit;
+    bottom: inherit;
+    left: inherit;
+    width: auto;
+    max-width: $govuk-page-width * 2 / 3;
+    height: auto;
+    margin: $govuk-modal-margin auto;
+    border: $govuk-border-width-form-element solid $govuk-input-border-colour;
+  }
+
+  @include govuk-font($size: 19);
+  @include govuk-text-colour;
+  @include govuk-focusable;
```

Comment:
Personally I'd put these three `@include` lines above the `@include govuk-media-query($from: tablet) {` line, because they're effectively properties that belong with the others. But this is non-blocking.